src/main/sensors/gyro.c

```diff
@@ -425,21 +425,21 @@ FAST_CODE void gyroUpdate(void)
     }
 
     if (active != 0) {
-        gyro.gyroADC[X] = adcSum[X] / active;
-        gyro.gyroADC[Y] = adcSum[Y] / active;
-        gyro.gyroADC[Z] = adcSum[Z] / active;
+        for (int axis = FD_ROLL; axis <= FD_YAW; axis++) {
```

Comment:
Why are you rolling up the loop?

Unrolled loop has better performance at the cost of slightly increased code size. But in such a short loop the code size advantage is minimal.

And, depending of compiler optimization flags, the compiler will probably unroll this loop anyway.